tools/c7n_org/c7n_org/cli.py

```diff
@@ -553,6 +554,13 @@ def run_account(account, region, policies_config, output_path,
             # Variable expansion and non schema validation (not optional)
             p.expand_variables(p.get_variables(account.get('vars', {})))
             p.validate()
+            normalized_resource = p.resource_type.replace('aws.', '')
+            aws_resource = clouds['aws'].resources.get(normalized_resource).resource_type.service
+            if region not in boto3.session.Session().get_available_regions(aws_resource):
```

Comment:
there's a couple of issues with this implementation

1. its very aws specific, c7n-org is used across gcp and azure providers
1. its duplicating functionality already in the c7n.resources.aws instead of reusing
1. the fetch of service region mappings can be fetched once per region instead of per policy

Reply:
Ok, I've updated my code and push a new version regarding your inputs.

Reply:
thanks!